orc8r/cloud/go/tools/swaggergen/generate/generate.go

```diff
@@ -165,6 +166,51 @@ func GenerateModels(targetFilepath string, configFilepath string, rootDir string
 	return nil
 }
 
+// GenerateStandaloneSpec generates a standalone spec file for a particular
+// Swagger spec.
+func GenerateStandaloneSpec(targetFilepath string, specs map[string]MagmaSwaggerSpec, outPath string) error {
+	absTargetFilepath, err := filepath.Abs(targetFilepath)
+	if err != nil {
+		return errors.Wrapf(err, "target filepath %s is invalid", targetFilepath)
+	}
+
+	var yamlSpecs []string
+	var yamlCommon string
+	for specFilepath, s := range specs {
+		if specFilepath != absTargetFilepath {
+			// Clearing fields not utilized by target service
```

Comment:
This comment describes "what" -- but when we're adding an inline comment, we generally want to describe "why"